FormidableLabs/groqd
# GroqD 1.0 Prep - Updated Arcade Samples

Updated the Arcade with v1.0 samples

# Testing
1. Load this branch's Arcade: https://groqd-git-groqd-10-arcade-samples-formidable-labs.vercel.app/arcade

2. At the top, load one of the samples:
<img width="692" alt="image" src="https://github.com/user-attachments/assets/8338c1a3-91d0-4251-9c6b-df9e2f394558" />

3. Verify that the query executes correctly.
4. Repeat for each sample query
5. Note: there is 1 query that is designed to fail, called <kbd>Basic Query with VALIDATION ERRORS</kbd>:
<img width="1757" alt="image" src="https://github.com/user-attachments/assets/8d6424a0-d7d9-48c0-a9de-35a4b3fbbcf2" />
